packages/node/src/index.ts

```diff
@@ -1,5 +1,208 @@
+import type { GetProjectResponse200 } from './.api/apis/developers';
+import type { Options } from './lib/log';
+import type { NextFunction, Request, Response } from 'express';
+
+import readmeSdk from './.api/apis/developers';
+import findAPIKey from './lib/find-api-key';
+import { getGroupIdByApiKey } from './lib/get-group-id';
 import { getProjectBaseUrl } from './lib/get-project-base-url';
 import { log } from './lib/log';
+import { buildSetupView } from './lib/setup-readme-view';
+import { testVerifyWebhook } from './lib/test-verify-webhook';
 import verifyWebhook from './lib/verify-webhook';
 
-export { log, verifyWebhook, getProjectBaseUrl };
```

Comment:
Does losing `log` mean that we're no longer going to be able to support non-Express frameworks? All of our non-Express examples in https://github.com/readmeio/metrics-sdks/tree/main/packages/node/examples are going to break once we publish a new release to `readmeio`.

Reply:
That export is still there! This should be fully backwards compatible: https://docs.readme.com/main/docs/unified-snippet-docs#/